src/rules/converters/no-async-without-await.ts

```diff
@@ -0,0 +1,11 @@
+import { RuleConverter } from "../converter";
+
+export const convertNoAsyncWithoutAwait: RuleConverter = () => {
+    return {
+        rules: [
+            {
+                ruleName: "require-await",

```

Comment:
Wait, I just noticed - this should be:

```suggestion
                ruleName: "@typescript-eslint/require-await",
```

Reply:
Hi, I develop the suggested modification and pull new commits, but the tests end-to-end has failed :(